fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/api/GlobalConfigurationApiResource.java

```diff
@@ -76,29 +76,44 @@ public GlobalConfigurationApiResource(final PlatformSecurityContext context,
     @GET
     @Consumes({ MediaType.APPLICATION_JSON })
     @Produces({ MediaType.APPLICATION_JSON })
-    public String retrieveConfiguration(@Context final UriInfo uriInfo,@DefaultValue("false") @QueryParam("survey") final boolean survey) {
+    public String retrieveConfiguration(@Context final UriInfo uriInfo,
```

Comment:
I don't know whether you have followed our discussion. Let me put it here what I understood. You are supposed to add a new method in read platform service. Not in the API resource. Let me know if you differ this :-)